projects/fields/README.md

```diff
@@ -10,17 +10,18 @@
   * `g` - Euclidean distance with `gdal.FillNodata()`
   * `n` - none
 * `--aggregation-method`, default: `mean`
-  * `min`, `max`, `mean`, `max_minus_min`
-* `--year-aggregation-method`, default: 0
-  * `0`: do nothing
-  * `1`: aggregate deviations with same year
+  * `min`, `max`, `mean`, `max_minus_min`, `median`, `quantile_0.4`
+* `--year-aggregation-method`, default: `none`
+  * `none`: do nothing
+  * `min`, `max`, `mean`, `max_minus_min`, `median`: aggregate deviations with same year
 * `--dilation-method`, default: 3
   * `1`: images
   * `2`: deviations
   * `3`: final deviation
 * `--deviation-method`, default: 1
-  * `0`: do nothing
-  * `1`: subtract mean NDVI
+  * `raw`: do nothing
+  * `subtract_mean`: subtract mean NDVI
+  * `quantile`: calculate quantile of each pixel's value
```

Comment:
Is this `quantile` implemented in `compute_deviation`? Or it is named `cdf`? `cdf` is a too short name.